wiki/Beatmaps/Genre_and_language/en.md

```diff
@@ -0,0 +1,48 @@
+---
+tags:
+  - genres
+  - languages
+  - metadata
+---
+
+# Genre and language
+
+![A screenshot of source, genre, language, and tags on a beatmap page](img/beatmap-info.png "Genre and language are listed on beatmap pages.")
+
+[Beatmaps](/wiki/Beatmaps) submitted to the website have **genre** and **language** fields to help categorise songs. They default to `Unspecified` and `Other` respectively, and can be edited by [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) and [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) members. Mapset owners are also able to set the genre and language of their beatmaps as long as the beatmap's status is WIP<!-- TODO: link -->, [Pending](/wiki/Beatmaps#pending), or [Graveyard](/wiki/Beatmaps#graveyard).
+
+Genre and language have no function in the game, but will be available as search options in osu!lazer<!-- TODO: link me -->.
+
+## Beatmap listing
+
+![The available filters on the beatmap listing](img/beatmap-listing.png)
+
+The [beatmap listing](https://osu.ppy.sh/beatmapsets) has filters available for genre and language in the `More Search Options` dropdown.
+
+## List of genres
+
+| Genre | Traits of applicable songs |
+| :-- | :-- |
+| Unspecified | No songs are applicable. This genre is a placeholder until a proper one is set. |
+| Video Game | Made for or made popular by a video game. This is also used for arrangements and remixes of those songs. |
+| Anime | Made for or made popular by an anime or similar medium. |
+| Rock | Mostly focusing on guitars, drums, and bass, typically "built on a foundation of simple unsyncopated rhythms" and characterised by "live performance and a focus on serious and progressive themes" ([*Rock music* on Wikipedia](https://en.wikipedia.org/wiki/Rock_music#Characteristics)). |
+| Pop | Prominent in pop culture, and composed to be catchy by use of simple structure and repeated choruses or melodies. |
+| Other | Not belonging in the other genres. |
```

Comment:
Could make this "Not belonging to any of the listed genres" ? Is more specific to this listing that way imo